gh_pages/_source/session3/ros2/4-Motion-Planning-RVIZ.md

```diff
@@ -1,6 +1,7 @@
 # Motion Planning using RViz
 > In this exercise, we will (finally) learn how to use the MoveIt! RViz plugin to plan and execute motion on a simulated robot. We will explore the different options and constraints associated with both MoveIt! and the RViz plugin.
 
+
```

Comment:
Extra line of whitespace? I believe .md files will edit this out / flatten it anyway, but seems like an easy fix to not add it in the first place.